realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -271,6 +277,9 @@ public final boolean load() {
      * Removes a previously registered listener.
      *
      * @param listener the instance to be removed.
+     * @throws IllegalArgumentException if the change listener is {@code null}.
+     * @throws IllegalArgumentException if object is an un-managed RealmObject.
```

Comment:
1. `if object is an un-managed` -> `if the object is an unmanaged` 

2. Can we combine line 280 and 281 in the same line?
```
* @throws IllegalArgumentException if the change listener is {@code null} or the object is an unmanaged RealmObject.
```